lib/matplotlib/axes/_base.py

```diff
@@ -1518,6 +1521,9 @@ def axis(self, *v, **kwargs):
             Passed to set_{x,y}lim functions, if observers
             are notified of axis limit change
 
+        y_bound : bool
```

Comment:
Isn't it `ylim_bound`? Also, this name doesn't really imply what it does to me, though I'm not sure of a better name at this point.